cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -29,7 +29,7 @@ type Exception struct {
 
 var (
 	// Lists to memorize results required for the subsequent steps
-	ecsTaskList   = []ecs.Task{}
+	EcsTaskList   = []ecs.Task{}
```

Comment:
Not sure what you mean. Unfortunately the instance steps are already coupled to the task steps, as you need to start a task to get an instance event in the event stream.